src/schema/v1/show.ts

```diff
@@ -664,16 +665,20 @@ const Show: GraphQLFieldConfig<void, ResolverContext> = {
       description: "The slug or ID of the Show",
     },
   },
-  resolve: (_root, { id }, { showLoader }) => {
+  resolve: (_root, { id }, { showLoader, accessToken }) => {
+    const decodedJwt = decodeArtsyJWT(accessToken)
+    const partnerIds: Array<String> = decodedJwt ? decodedJwt.partner_ids : []
+    const isAdmin: boolean =
+      decodedJwt && decodedJwt.roles.split(",").includes("admin")
```

Comment:
So seeing as we know this is an admin, perhaps then it would make sense to explicitly use the authenticated loader only for these cases (and have all other requests cached)?